backend/server.py

```
@@ -71,14 +138,65 @@ def calc():
+        pass
+
+
+@app.get("/stop-vdb-update", status_code=200)
+async def stop_vdb_update(logto_token: str):
+    if not user_is_admin(logto_token):
+        raise HTTPException(400)
+    
+    try:
+        os.remove(VDB_LOCK)
+
+    except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)